tests/puppeteer.js

```diff
@@ -0,0 +1,47 @@
+const server = require('./server');
+const puppeteer = require('puppeteer');
+const assert = require('assert');
+const PORT = 8080;
+
+/**
+ * Puppeteer init method
+ * @param {function} callback
+ */
+async function init(callback) {
+  const browser = await puppeteer.launch({});
+
+  const page = await browser.newPage();
+  await page.goto(`http://localhost:${PORT}`, {
+    waitUntil: 'networkidle0',
+  });
+
+  /**
+   * FID value is returned
+   * @return {number}
+   */
+  function inputDelayHandler() {
+    return new Promise((resolve, reject) => {
+      perfMetrics.onFirstInputDelay(function(delay, event) {
+        resolve(delay);
+      });
+
+      /**
+       * Have to handle all events here ie click, touchstart
+       * Currently only mousedown is handled
+       */
+      var mEvent = document.createEvent('MouseEvents');
```

Comment:
I believe you can use `new MouseEvent('mousedown')` rather than `document.createEvent()` here.